AzureMonitorAgent/agent.py

```diff
@@ -2132,4 +2170,4 @@ def get_error_message(self, operation):
                                                                    self.error_code)
 
 if __name__ == '__main__' :
-    main()
+    main()
```

Comment:
Just noticed this after signing off. Looks accidental change (line #2173)

Reply:
fixed this